src/tools/annotation/ProbeTool.js

```diff
@@ -62,6 +75,12 @@ export default class ProbeTool extends BaseAnnotationTool {
           highlight: true,
           active: true,
         },
+        textBox: {
```

Comment:
The "Probe" annotation tool does not have the "textBox" structure. I added textBox is just to enable or disable. There was a strange problem, if you make just one click he does not draw, has to drop and draw. Without textBox structure, it works right.